### PR TITLE
When selecting/deselecting networks on the chain list screen, enable/disable changes are not saved if the new chain is saved #3420

### LINKED ISSUE #3420: When selecting/deselecting networks on the chain list screen, enable/disable changes are not saved if the new chain is saved
When in the enabled network screen, a user selects / deselects one or more networks. Then the user deletes a custom network or adds a custom network. The changes made to the selections are not preserved.

The following instructions assume the Expanse Network has not been previously added.

1. Tap the settings tab button when on the Home Screen.
2. Tap the select active networks table row.
3. Tap on  some of the selected networks to deselect them.
4. Tap on some of the unselected networks to select them.
5. Tap the + button in the upper right hand corner to add a network.
6. Type "Expanse Network" in the network name field.
7. Type "https://node.expanse.tech" in the RPC URL field.
8. Type "2" in the Chain ID field.
9. Type  "EXP" in the Symbol field.
10. Type "https://expanse.tech" in the Block Explorer URL field.
11. Tap "Save network" button. 
12. When "Warning" alert pops up hit continue.
13. After a few moments, the home screen will be shown with the wallet tab selected.
14. Tap settings tab button 
15. Tap select active networks

**Expect**
16a. Selections made to networks be changed. 

**Observed**
16.b Selections made to networks were not changed.

Version 3.29 Build 408

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Fixes #3420

### CODE REVIEW

**Comment by vladyslav-iosdev** [comment]
@hboon this is ready for review

File: AlphaWallet/InCoordinator.swift [hunk 1]
```diff
@@ -721,6 +721,9 @@ class InCoordinator: NSObject, Coordinator {
                 Config.setChainId(server.chainID)
             case .loadUrlInDappBrowser:
                 break
+            case .reloadServers(let servers):
+                var c = config
```
[the remark this answers is not in the document]
Comment (vladyslav-iosdev) [reply AlphaWallet/InCoordinator.swift:725]: i am not clearly understand whe do we need it here? 
there is no `restartQueue.remove(each)` line for case `reloadServers `

[the remark this answers is not in the document]
Comment (hboon) [reply AlphaWallet/InCoordinator.swift:725]: Sorry, I mean. it needs to add that line too, so it becomes:

```
case .loadUrlInDappBrowser:
    break
case .reloadServers(let servers):
    restartQueue.remove(each)
    var c = config
    c.enabledServers = servers
}
```

[the remark this answers is not in the document]
Comment (vladyslav-iosdev) [reply AlphaWallet/InCoordinator.swift:726]: oh, yep, right.

**Comment by hboon** [comment]
@vladyslav-iosdev readying to merge?

**Review by hboon — APPROVED** [review]

**Pull request merged**